source/includes/code-examples/FindOneAsync.cs

```diff
@@ -0,0 +1,82 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindOneAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find one document using builders
+        var buildersDocument = FindOneRestaurantBuilderAsync().Result.ToBsonDocument();
+        WriteLine("Finding a document with builders...");
+        WriteLine(buildersDocument);
+
+        // Extra space for console readability
+        WriteLine();
+
+        // Find one document using LINQ
+        var linqDocument = FindOneRestaurantLINQAsync().Result.ToBsonDocument();
+        WriteLine("Finding a document with LINQ...");
+        WriteLine(linqDocument);
+    }
+
+    private static async Task<Restaurant> FindOneRestaurantBuilderAsync()
+    {
+        // start-find-builders
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Name, "Bagels N Buns");
+
+        return await _restaurantsCollection.Find(filter).FirstOrDefaultAsync();
```

Comment:
Should we use `FindAsync` in Async example?

Reply:
nope, I think I asked it somewhere before. Find (fluent) and FindSync/FindAsync are different approaches and as I recall at least initially we wanted to mention only fluent approach